docs/readme.md

```diff
@@ -1 +1,124 @@
-readme.file
+# Chicken & Fries
+
+This is a project for the Web Application Integration online class at Full Sail University
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine.
+
+### Prerequisites
+
+To run this project on you local computer you need to have NodeJs and NPM installed on your machine
+
+### Installing
+
+A step by step series of examples that tell you how to get a development env running
+
+First clone or download the files on your machine
+
+Place the files in an easy to access folder on your machine
+
+```
+Desktop
+```
+Open your terminal window and run the following command
+```
+nodemon bin/www
+```
+Open your favorite browser and go to
+```
+localhost:3000
+```
+
+#Table Of Contents
```

Comment:
You'll want to fill in these links in the table of contents to lead to the sections of your project spec.